PKG-INFO

```diff
@@ -1,6 +1,6 @@
 Metadata-Version: 1.1
 Name: pybird
-Version: 1.0.10
+Version: 1.1.1.2+stackpath
```

Comment:
This version doesn't comply with PEP-440, any reason why? https://www.python.org/dev/peps/pep-0440/

Reply:
Scratch that, I see this fits into the local versioning scheme. Dunno if this is valid for PKG-INFO though..

Reply:
It originally complained because I was using "-stackpath", "+stackpath" appeared to pass PEP-440 (or at least it stopped complaining). I wasn't sure how to approach the versioning here to say that it's "stackpath" while also having it be correct for PKG-INFO. Will dig more